kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiCommands.java

```diff
@@ -45,7 +45,7 @@ public boolean sql(String line) {
 
   /** Extract and clean up the first command in the input. */
   private String getFirstCmd(String cmd, int length) {
-    return cmd.substring(length);
+    return cmd.substring(length).trim();
```

Comment:
This should be to avoid submitting Python code to remove indentation and cause syntax failure.  
https://github.com/apache/incubator-kyuubi/pull/3762

Wouldn't it be better if we could match different commands to determine trim?

cc @cfmcgrady  @pan3793

Reply:
> This should be to avoid submitting Python code to remove indentation and cause syntax failure.

The `getFirstCmd` method is called only in `sourceFile`, I don't understand why it is affected?

Reply:
> > This should be to avoid submitting Python code to remove indentation and cause syntax failure.
> 
> The `getFirstCmd` method is called only in `sourceFile`, I don't understand why it is affected?

seems you are reasonable

also cc @bowenliang123, can you help verify this change?

Reply:
tested locally, this change does not affect submitting python code.